verification/testNF90io/input/gendata.py

```diff
@@ -0,0 +1,94 @@
+from numpy import *
+from scipy import *
+from pylab import *
+from shutil import copy
+import numpy.matlib as matlib
```

Comment:
Are all of these imports necessary for the gendata script? If not, they could be removed to reduce the python package dependencies.

Also note that `import *` is not recommended since it confuses the namespace. Better to `import numpy as np`